---
title: "**Mid point report**"
author: "**Muhammad Ilyas**"
format:
  pdf:
    toc: true
    toc-depth: 3
    toc-title: Contents
    number-sections: false
    colorlinks: true
    documentclass: report
    papersize: A4
    latex_engine: xelatex
    mainfont: Times New Roman
    geometry:
      - top=2cm
      - left=2cm
      - bottom=2cm
      - right=2cm
execute:
  echo: true
  eval: false
engine: knitr
date: today
bibliography: 'https://api.citedrive.com/bib/c06b758b-f2c7-4246-9fa0-4de91852b333/references.bib?x=eyJpZCI6ICJjMDZiNzU4Yi1mMmM3LTQyNDYtOWZhMC00ZGU5MTg1MmIzMzMiLCAidXNlciI6ICIyMzkxIiwgInNpZ25hdHVyZSI6ICJjYjFkNjVkNDgxZmY3YzFmYjI0MWY2MjNiZWU0MWM0YjRjZWNhODRjODNmMTE4YmM1MTQ0OTk4YjlmYTA3ZDQ1In0=/bibliography.bib'
csl: plos.csl
editor: visual
---

# Research Questions

## Differential gene expression

## Relevancy to research

# Releant Literature

## Studies on differential gene expression

## Studies on Japanese quail

# Data

## Quail transcriptome data

Provided by Dr. Barbara's Lab from Centre of Ecology and Conservation. The experiment was designed to investigate the effect of reproductive investment on animal physiology. Two lines with high and low investment based on egg size were selected to find up-regulation or down-regulation of genes. Both the lines were provided enough food to control for food related stress. The tissues investigated were brain, follicles and liver.

# Data Wrangling

The raw data needs processing to before any analysis can be performed. A pipeline was designed to process the raw transcriptome data. The process involves quality check, trimming adapter sequences and discarding low quality reads, mapping the transcriptome against a reference genome and obtaining count tables. The count tables can be used for down stream analysis of the data to find out which genes are differentially expressed. Due to the large size of data the servers were used for carrying the process. Most of the analysis was performed in R using Bioconductor, except for the mapping and generating count tables which were done using STAR and HTSeq respectively. The fastp [@Chen_2018] was used to do the quality control and adapter trimming.

```{bash}
cd dissertation_project/data 
```

# Next Steps

```{r, eval=TRUE, echo=FALSE, warning=FALSE}
#| label: gant-chart
#| fig-cap: "Gant chart for next steps"
#| fig-alt: "Gant chart for next steps"
#| message: false
#| warning: false
#| fig-height: 6
#| fig-width: 10
#| fig-dpi: 900
#| fig-align: center


# Loading the necessary pacakges
library(plan)

g <- new("gantt")
g <- ganttAddTask(g, "Data Analysis and Modelling")
g <- ganttAddTask(g, "Further Data Wrangling","2023-06-16","2023-06-30",done=0 )
g <- ganttAddTask(g, "Finalize questins","2023-06-25","2023-07-05",done=0 )
g <- ganttAddTask(g, "Review of literature","2023-06-27","2023-07-10",done=0 )
g <- ganttAddTask(g, "Data Modelling")
g <- ganttAddTask(g, "Classical approaches","2023-07-06","2023-07-15",done=0 )
g <- ganttAddTask(g, "Machine-Learning approaches","2023-07-16","2023-07-31",done=0 )
g <- ganttAddTask(g, "Write-up")
g <- ganttAddTask(g, "Writing","2023-07-25","2023-08-10",done=0 )
g <- ganttAddTask(g, "Supervisor feedback","2023-08-11","2023-08-20",done=0 )
g <- ganttAddTask(g, "Get Feedback and Update","2023-08-20","2023-08-23",done=0 )
g <- ganttAddTask(g, "Final Submission")

#png("output/gantt_chart.png", width = 12, height = 8, res = 900, units = "in")


plot(g, ylabel=list(font=ifelse(is.na(g[["start"]]), 2, 1)),
     event.time="2023-08-24", event.label="Submission Date", col.event=c("red"),
     main = "")

```

# References

